README.md

```diff
@@ -84,11 +84,11 @@ properties, e.g.:
 reformat or transform networks into networks with other properties,
 e.g.:
 
--   `to_blocks()`, `to_edges()`, `to_giant()`, `to_matching()`,
-    `to_mode1()`, `to_mode2()`, `to_multilevel()`, `to_named()`,
-    `to_onemode()`, `to_redirected()`, `to_simplex()`, `to_subgraph()`,
-    `to_ties()`, `to_twomode()`, `to_undirected()`, `to_uniplex()`,
-    `to_unnamed()`, `to_unsigned()`, `to_unweighted()`
+-   `to_blocks()`, `to_edges()`, `to_giant()`, `to_main_component()`,
```

Comment:
Should `to_edges` and `to_main_component` still be here?

Reply:
Possibly because defunct means still exported